demo.rb

```diff
@@ -0,0 +1,21 @@
+require './lib/goods.rb'
```

Comment:
Nice demo! 💥

Reply:
This demo could be turned into a feature test if you wrote an assertion at the bottom